build/alpine/install-packages.sh

```diff
@@ -1,39 +1,22 @@
 #!/bin/sh
 
 set -e
-set -o pipefail
 
+# Install necessary packages
 apk add --no-cache -U \
-    openssl \
-    imagemagick \
-    file \
-    lsof \
-    su-exec \
-    coreutils \
-    findutils \
-    procps \
-    shadow \
-    bash \
-    curl iputils \
-    git \
-    jq \
-    mysql-client \
-    tzdata \
-    rsync \
-    nano \
-    sudo \
-    tar \
-    zstd \
-    nfs-utils \
-    libpcap \
-    libwebp \
-    libcap
+    openssl imagemagick file lsof su-exec coreutils findutils procps shadow bash \
+    curl iputils git jq mysql-client tzdata rsync nano sudo tar zstd nfs-utils \
+    libpcap libwebp libcap
```

Comment:
Please revert this. It must be one per line for readability and maintainability.

Reply:
Sorry about that reverted back.